codespell_lib/data/dictionary.txt

```diff
@@ -10064,6 +10071,7 @@ oterwice->otherwise
 oterwise->otherwise
 oterwize->otherwise
 othe->other
+other wise->otherwise
```

Comment:
whitespace

Reply:
removed

Reply:
@Gelma the commit text needs to be amended as well 
![image](https://user-images.githubusercontent.com/4140247/54990900-308edf80-4f92-11e9-9d0e-6cb257fd334a.png)